packages/angular-material/test/vertical-layout.spec.ts

```diff
@@ -25,109 +25,113 @@
 import { NgRedux } from '@angular-redux/store';
 import { MockNgRedux } from '@angular-redux/store/lib/testing';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { JsonFormsOutlet } from '@jsonforms/angular';
+import { JsonFormsOutlet, UnknownRenderer } from '@jsonforms/angular';
 import { UISchemaElement, VerticalLayout } from '@jsonforms/core';
+import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/testing';
 import {
-  VerticalLayoutRenderer,
-  verticalLayoutTester
+    VerticalLayoutRenderer,
+    verticalLayoutTester
 } from '../src/layouts/vertical-layout.renderer';
 
 describe('Vertical layout tester', () => {
-  it('should succeed', () => {
-    expect(verticalLayoutTester({ type: 'VerticalLayout' }, undefined)).toBe(1);
-  });
+    it('should succeed', () => {
+        expect(verticalLayoutTester({ type: 'VerticalLayout' }, undefined)).toBe(1);
+    });
 });
 describe('Vertical layout', () => {
-  let fixture: ComponentFixture<any>;
-  let component: any;
+    let fixture: ComponentFixture<any>;
+    let component: any;
 
-  beforeEach(() => {
-    TestBed.configureTestingModule({
-      declarations: [
-        VerticalLayoutRenderer,
-        JsonFormsOutlet
-      ],
-      imports: [],
-      providers: [
-        { provide: NgRedux, useFactory: MockNgRedux.getInstance }
-      ]
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                VerticalLayoutRenderer,
+                UnknownRenderer,
+                JsonFormsOutlet
+            ],
+            imports: [],
+            providers: [
+                { provide: NgRedux, useFactory: MockNgRedux.getInstance }
+            ]
+        }).overrideModule(BrowserDynamicTestingModule, {
+            set: {
+                entryComponents: [
+                    UnknownRenderer
+                ]
+            }
+        }).compileComponents();
+        MockNgRedux.reset();
+        fixture = TestBed.createComponent(VerticalLayoutRenderer);
+        component = fixture.componentInstance;
     });
-    TestBed.compileComponents();
-    MockNgRedux.reset();
-  });
-
-  beforeEach(() => {
-    fixture = TestBed.createComponent(VerticalLayoutRenderer);
-    component = fixture.componentInstance;
-  });
 
-  it('render with undefined elements', () => {
-    const uischema: UISchemaElement = {
-      type: 'VerticalLayout'
-    };
-    const mockSubStore = MockNgRedux.getSelectorStub();
-    component.uischema = uischema;
+    it('render with undefined elements', () => {
```

Comment:
all three tests are also similar.
only group needs a 4 test to check for the correct label.